tarsier/tag_utils.ts

```diff
@@ -174,7 +179,7 @@ function getElementXPath(element: HTMLElement | null) {
 
     let sibling = element.previousElementSibling;
     while (sibling) {
-      if (sibling.tagName === element.tagName) {
+      if (sibling.tagName === element.tagName && sibling.id != tarsierId) {
```

Comment:
If we add a tarsier tag span, it was affecting the index of the xpath (the tag would be for `span[4]` instead of `span[3]` for example